stories/organisms/components/filePicker/FilePicker.stories.mdx

```diff
@@ -97,6 +97,36 @@ It is a combination of the [FileInput](/docs/atoms-fileinput--overview) and the
 
 <ArgsTable story="Overview" />
 
+## Clear all
```

Comment:
Please, update the story according to the new `showClearAll` property.

Reply:
I think the title should be a bit more contextualized. For me, the title should clearly evoke the possibility or the effect that it is possible to obtain from the component. As already mentioned by @fredericvilcot: we have to adopt a content marketing approach for writing stories.

Reply:
Ok I understand. So what title can we put ? "Show clear all" is not enough I think.
What do you think @fredericvilcot @ccamel

Reply:
I will keep Show Clear All as a title, what do you think @ccamel ?

Reply:
Go with it. we can be more attractive if needed in a second time.